### DISCUSSION TITLE
Lose Settings and API Keys when Upgrading Past 2021.4.7192

### DISCUSSION THREAD

**Discussion by mason-mcglothlin** [datalust/seq-tickets#2006] [Help]
Our Seq instance on Windows Server was running on v2021.2.5647. We tried to update straight to the latest version by installing the MSI for v2023.4.10219, but all our settings and API keys disappeared after the install completed.

I managed to uninstall Seq v2023.4.10219 via Programs and Features and then re-run the installer for v2021.2.5647 and our settings and API keys came back. So I tried to update to several different versions, the most recent version I was able to update to and keep all our settings and API keys was v2021.4.7192. So that's the version I have left us on for now.

Updating to any of these results in loss of settings and API keys:
2022.1.7378 (next version and oldest v2022.x)
2022.1.7929 (latest 2022.x)
2023.4.10146 (latest 2023.x)

Our subscription ID is SC4EF4MXAEIXQ

The server does not have Internet access. Please let me know if there's any data I need to gather for you or steps to try.

**Comment by nblumhardt** [comment]
Hi Mason, thanks for reaching out. That's very troubling!

There should be some details of what happened around the time of the upgrade in C:\ProgramData\Seq\Logs - if you're able to send the log file/entries from slightly before, to slightly after, the unsuccessful upgrade, we'll dig in and figure it out.

Best regards,
Nick

**Reply by nblumhardt** [reply]
Sorry I should add - it'll be easiest to work through this via our `support@datalust.co` email address, if you can follow up there. Thanks!
